arangodb/tests/docker/arangodb3/arangod.conf

```diff
@@ -43,7 +43,8 @@ statistics = true
 # gid = arangodb
 
 
-
+[query]
```

Comment:
This sets all queries to be slow queries, so that the dashboard can populate the slow query widget out of the box when testing.

Reply:
Is there a way to leave a comment on this file about that?

Reply:
Yep! I can add a comment here.